setup.py

```diff
@@ -26,6 +28,12 @@ def _read_me() -> str:
     return readme
 
 
+def _requirements() -> List[str]:
+    file = Path((__file__)).parent.joinpath('requirements.txt')
+    with open(file, 'r') as fh:
+        return fh.readlines()
```

Comment:
I think we should make a couple of tweaks here:

* Let's add a `"""` docstring, kind of like `_read_me()` has.
* Drop the extra parens in `Path((...))`.
* Let's use `Path(__file__).parent / 'requirements.txt'`, as I think that's more idiomatic [with pathlib](https://docs.python.org/3/library/pathlib.html#operators).
* IMO we should drop the `r` on `open()`, but explicitly specify `encoding='utf-8'`.
* Tweak the code to make it a bit more robust with blank and comment lines, per below:

```python
with open(Path(__file__).parent / 'requirements.txt') as fh:
	reqs = []
	for line in fh:
		line = line.strip()
		if not line or line.startswith('#'):
			continue
		reqs.append(line)
        return reqs
```

Reply:
yeah not sure what happened here, perhaps I forgot to push the commit. 

I also came across https://setuptools.pypa.io/en/latest/pkg_resources.html#requirements-parsing but I've kept it for another PR with a `TODO` note for now.